src/api/app/controllers/webui/user_controller.rb

```diff
@@ -73,14 +74,17 @@ def save
     @displayed_user = User.find_by_login(params[:user][:login])
 
     unless User.current.is_admin?
-      if User.current != @displayed_user
+      if User.current != @displayed_user || !@configuration.accounts_editable?
         flash[:error] = "Can't edit #{@displayed_user.login}"
         redirect_back(fallback_location: root_path)
         return
       end
     end
 
-    @displayed_user.assign_attributes(params[:user].slice(:realname, :email).permit!)
+    unless @configuration.ldap_enabled?
```

Comment:
For admins yes but for normal users it might make sense to render a 404 / 403 etc?